translations/mod-fqm-manager/en.json

```
@@ -308,5 +308,20 @@
   "entityType.drv_holdings_record_details.holdings_temporary_location": "Holdings temporary location",
   "entityType.drv_holdings_record_details.holdings_temporary_location_id": "Holdings temporary location ID",
   "entityType.drv_holdings_record_details.holdings_statistical_code_ids": "Holdings statistical code ID list",
-  "entityType.drv_holdings_record_details.holdings_statistical_codes": "Holdings statistical code list"
+  "entityType.drv_holdings_record_details.holdings_statistical_codes": "Holdings statistical code list",
+
+  "entityType.src_organization_type": "Organization types",
+  "entityType.src_organization_type.id": "ID",
+  "entityType.src_organization_type.organization_types_name": "Type",
+  "entityType.drv_organization_details" : "Organizations - vendor info",
+  "entityType.drv_organization_details.last_updated": "Org last updated",
```

We should probably use the full name, "Organization"
```suggestion
  "entityType.drv_organization_details.last_updated": "Organization last updated",
```